source/partials/replace-credit-card.md

```diff
@@ -0,0 +1,9 @@
+The **Replace Card** button allows you to change the credit card to which a site is being billed, or to update an existing card's billing or expiration information.
+
+1. From the User Dashboard, click the **<span class="glyphicons glyphicons-cogwheel"></span> Account** tab and select **Billing** from the left side.
+
+1. In the box that appears, check the box next to **Yes, I want to continue** and click **Continue**.
```

Comment:
Is "box" the right term? Is this a modal? Can we be more descriptive about which item on the page we're referring to?

Sorry, if I had a user with paid sites I'd look for myself.

Reply:
it's a modal, but there's an entire step or two that vanished from here. fixing